app/models/patient.rb

```diff
@@ -493,6 +520,47 @@ def latest_transfer
   }
   # rubocop:enable Style/MultilineBlockChain
 
+  # Patients in the exposure workflow have finished their monitoring period IF:
+  # - not in continuous exposure
+  #    AND EITHER
+  # - last exposure date is on or after (today - 'monitoring_period_days') in patient local time
+  #    OR
+  # - last exposure date is null and created date is on or after (today - 'monitoring_period_days') in patient local time
+  scope :end_of_monitoring_period, lambda {
```

Comment:
As part of including and using this scope, I think we can remove the `end_of_monitoring_period?` method that is there now since it will no longer be used.

Reply:
Removing this method